components/dropdown/test/dropdown-menu.test.js

```diff
@@ -21,6 +22,19 @@ const itemFixture = html`
 	</div>
 `;
 
+const linkFixture = html`
+	<div>
+		<d2l-dropdown>
+			<button class="d2l-dropdown-opener">Open it!</button>
+			<d2l-dropdown-menu id="dropdown">
+				<d2l-menu label="Helpful Links">
+					<d2l-menu-item-link text="menu.js (View)" href="../../menu/menu.js" id="first-item"></d2l-menu-item-link>
```

Comment:
Maybe the following test failure is related to this line opening content on new page:
```
Edge 18.18363 (Windows 10) ERROR
  Some of your tests did a full page reload!
04 11 2020 13:11:35.082:ERROR [reporter.browserlabs]: ✖ Test Errored
```

Thoughts?

Reply:
Yeah I think the `click()` performed in the test will cause a navigation to happen for this test. For the purpose of this test, I think we should be able to use either `href="#"` (which scrolls to the top of the page) or `href="javascript:void(0)"` (which does nothing).

Reply:
Agree with Dave here -- those options should work.